README.md

```diff
@@ -4,7 +4,8 @@
 About the Plugin
 ----------------
 
-apigee-smatdocs-maven-plugin is a utility for creating API models and rendering the OpenAPI Specficiation to Smart docs in the Apigee Developer Portal
+apigee-smatdocs-maven-plugin is a utility for creating API models and pushing an OpenAPI spec document into the API Catalog module for a Drupal 8-based developer portal, where it can then be rendered into documentation using SmartDocs or another renderer.
```

Comment:
```suggestion
apigee-smartdocs-maven-plugin is a utility for creating API models and pushing an OpenAPI spec document into the API Catalog module for a Drupal 8-based developer portal, where it can then be rendered into documentation using SmartDocs or another renderer.
```